src/scripts/browserDeviceProfile.js

```diff
@@ -688,7 +704,10 @@ import browser from './browser';
         }
 
         if (canPlayHls() && options.enableHls !== false) {
-            if (hlsInFmp4VideoCodecs.length && hlsInFmp4VideoAudioCodecs.length && userSettings.preferFmp4HlsContainer() && (browser.safari || browser.tizen || browser.web0s)) {
+            const nativeFmp4 = (browser.iOS && browser.iOSVersion >= 11) || browser.osx || browser.tizen || browser.web0s;
```

Comment:
In webOS 1.2 emulator, fMP4 doesn't seem to work. webOS 3 also doesn't seem to support fMP4 (https://github.com/shaka-project/shaka-packager/issues/915). I couldn't find fMP4 in LG's documentation.

Samsung specifies fMP4 for Tizen 5.5 (https://developer.samsung.com/smarttv/develop/specifications/general-specifications.html), but it works on my Tizen 3 and 4. I can't say anything about Tizen 2.3/2.4.

So we need to check `tizenVersion >= 3` and `web0sVersion >= 3.5` (_you can omit checking `browser.tizen` and `browser.web0s` when checking version on `>=`_). Or we can wait for more info.